deps/rabbit/src/rabbit_khepri.erl

```diff
@@ -295,6 +295,21 @@ wait_for_leader(Timeout, Retries) ->
             throw(Reason)
     end.
 
+wait_for_register_projections() ->
+    wait_for_register_projections(retry_timeout(), retry_limit()).
+
+wait_for_register_projections(_Timeout, 0) ->
+    exit(timeout_waiting_for_khepri_projections);
+wait_for_register_projections(Timeout, Retries) ->
+    rabbit_log:info("Waiting for Khepri projections for ~tp ms, ~tp retries left",
+                    [Timeout, Retries - 1]),
+    try
+        register_projections()
+    catch
+        throw : {timeout, _ServerId} ->
```

Comment:
```suggestion
        throw : timeout ->
```

in Khepri 0.14.0+ this is simplified into just `{error, timeout}` https://github.com/rabbitmq/khepri/pull/256